apps/base/utils.py

```diff
@@ -228,3 +228,11 @@ def send_slack_notification(webhook=settings.SLACK_WEB_HOOK_URL, message=""):
         logger.exception(
             "Exception raised while sending slack notification. \n Exception message: {}".format(e)
         )
+
+
+def if_non_prod_aws_mocker(mocker):
```

Comment:
The name of the function can be more verbose I think, the current one is a bit unclear to me. Something like `mock_if_non_prod_aws(aws_mocker)` ?